src/core/ext/filters/client_channel/resolver/dns/event_engine/event_engine_client_channel_resolver.cc

```diff
@@ -327,7 +334,7 @@ void EventEngineClientChannelDNSResolver::EventEngineDNSRequestWrapper::
   // Do a subsequent hostname query since SRV records were returned
   for (auto& srv_record : *srv_records) {
     GRPC_EVENT_ENGINE_RESOLVER_TRACE(
-        "DNSResolver::%p Started resolving balancer hostname %s:%d",
+        "DNSResolver::%p Starting balancer hostname resolution for %s:%d",
         resolver_.get(), srv_record.host.c_str(), srv_record.port);
     balancer_hostname_handles_.insert(event_engine_resolver_->LookupHostname(
```

Comment:
Did you also want to add another log message after this to report the task handle?

Reply:
Sure, done.